conda_smithy/feedstock_content/ci_support/disabled.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="92" height="20">
```

Comment:
We probably need to move this so that it isn't copied by default into a feedstock, right?